src/core/Microsoft.PowerToys.Settings.UI/Views/GeneralPage.xaml.cs

```diff
@@ -40,6 +42,15 @@ public GeneralPage()
                 ShellPage.SendRestartAdminIPCMessage,
                 ShellPage.SendCheckForUpdatesIPCMessage);
 
+            ShellPage.ShellHandler.IPCResponseHandleList.Add((JsonObject json) =>
+            {
+                string version = json.GetNamedString("version");
+                if (version != string.Empty)
+                {
+                    ViewModel.LatestAvailableVersion = "Latest available version: " + version;
```

Comment:
localization?

Reply:
agreed, this type of string needs to be part of the localization.  I'd also suggest making the string be a var and then do a string.Format, this is more flexible for any dependent languages

Reply:
I would use two strings:
- "No update available."
- "New update available: v0.21.0"

Reply:
I updated the code, please take a look.

Reply:
> I updated the code, please take a look.

For me the code change looks good.

Can you please update the image in the PR details. I can't run your code, because I don't have the necessary infrastructure.

Reply:
@htcfreek sure, new images are in the summary.